Readme.md

```diff
@@ -11,20 +11,36 @@ Hermes is an application for cameras that can be used for detecting recyclable m
 * Python
 
 ### Available Scripts
-In the project directory, you can run:
-### `python3 scripts/docker.run.dependencies.py`
-The script starts the hermes-db dependency by using docker-compose.
+#### In the project directory, you can run:
+* python3 scripts/docker.run.dependencies.py
+    #### This script starts the hermes-db dependency by using docker-compose.
 
 ### Migrations Scripts
-In the project directory run:
-### `python3 scripts/migration.add.py --name Migration_Name`
-The script will create a new migration.
+#### In the project directory run:
+* python3 scripts/migration.add.py --name Migration_Name
 
-### `python3 scripts/migration.remove.py`
-The script will remove the last migration created. If migration was already applied to database you have to update first to the previous migration and then remove it.
+    #### This script will create a new migration.
 
-### `python3 scripts/migration.undo.py --name Migration_Name`
-The script will update database to the Migration_Name
+* python3 scripts/migration.remove.py
 
-### `python3 scripts/migration.apply.py`
-The script will apply the last migration created.
+    #### This script will remove the last migration created. If a migration was already applied to the database you have to first update it to the previous migration and then remove it.
+
+* python3 scripts/migration.undo.py --name Migration_Name
+
+    #### This script will update the database to the Migration_Name
+
+* python3 scripts/migration.apply.py
+
+    #### This script will apply the last migration created.
+
+### Abbreviations used:
+* *M*
+  * **M** comes from **Model** and it is used in **Business Layer**.
+* *P*
+  * **P** comes form **Parameter** and it is a **Parameter Model** used in **Business Layer**.
```

Comment:
Should change the form into `from` here ?

Reply:
Done